### PR TITLE
add created_by to all entities

### PR DESCRIPTION
**Pull request by pitr**
to facilitate debugging

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def sync_apps(entities, kio_url, access_token):
         entity['id'] = '{}[kio]'.format(app['id'])
         entity['application_id'] = app['id']
         entity['type'] = 'kio_application'
+        entity['created_by'] = 'zmon-entity-adapter'
```
Comment (vetinari) [inline app.py]: should this be a var / const?

Comment (pitr) [reply app.py]: addressed

**Comment by vetinari** [comment]
:+1:

**Comment by pitr** [comment]
:+1:

**Pull request merged**